rllib/examples/envs/env_connecting_to_rllib_w_tcp_client.py

```diff
@@ -0,0 +1,110 @@
+"""Example of running against a TCP-connected external env performing its own inference.
+
+The example uses a custom EnvRunner (TcpClientInferenceEnvRunner) to allow
+connections from one or more TCP clients to RLlib's EnvRunner actors, which act as
+RL servers.
+In this example, action inference for stepping the env is performed on the client's
+side, meaning the client computes all actions itself, applies them to the env logic,
+collects episodes of experiences, and sends these (in bulk) back to RLlib for training.
+Also, from time to time, the updated model weights have to be sent from RLlib (server)
+back to the connected clients.
+Note that RLlib's new API stack does not yet support individual action requests, where
```

Comment:
Why is this? Only because it is built in the `TcpEnvRunner` correct?

Reply:
Correct, this protocol (remote action inference on the RLlib server) is not supported yet. We need just another EnvRunner for that or build this logic into the existing one.